app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { useState, useEffect } from "react";
-import notesApi from "apis/notes";
 import { Button, PageLoader } from "neetoui";
-import EmptyState from "components/Common/EmptyState";
-import EmptyNotesListImage from "images/EmptyNotesList";
 import { Header, SubHeader } from "neetoui/layouts";
+
 import { useNotesState } from "contexts/notes";
+import EmptyState from "components/Common/EmptyState";
+import EmptyNotesListImage from "images/EmptyNotesList";
+import notesApi from "apis/notes";
+import sortOptions from "constants/sortOptions";
```

Comment:
Please add a line break here to separate relative/package imports and absolute imports.